### PR TITLE
Changed control scheme...

### PR DESCRIPTION
**Pull request by FusionLord**
- Changed controls;
- Merged mimic weapons;
- Moved scream to reload;
- Optimised some code;
- Removed temporary scream drawing;
- Added status bar for screaming;
- Removed unused swep;

### CODE REVIEW

File: gamemodes/attack_of_the_mimics/entities/weapons/swep_aotm_claws.lua [hunk 1]
```diff
@@ -117,7 +122,7 @@ function SWEP:PrimaryAttack()
 
 	-- effects
 	if tr.Hit then
-		self:EmitSound( self.Primary.HitSound[math.random(#self.Primary.HitSound)] )
+		self:EmitSound( table.Random(self.Primary.HitSound) )
```
Comment (raubana) [inline gamemodes/attack_of_the_mimics/entities/weapons/swep_aotm_claws.lua]: This was mentioned in the wiki to be less efficient. https://wiki.garrysmod.com/page/table/Random

File: gamemodes/attack_of_the_mimics/entities/weapons/swep_aotm_claws.lua [hunk 2]
```diff
@@ -174,31 +179,61 @@ function SWEP:SecondaryAttack()
...
 
 
-function SWEP:DrawHUD()
-	if not self:GetScreamReady() then
-		surface.SetDrawColor(Color(255,0,0))
+function SWEP:Reload()
+	if not self:GetScreamReady() then return end
+	
+	if SERVER then
+		self.Owner:EmitSound( table.Random(self.Scream.Sounds), 95, Lerp(math.random(), 80, 100) )
```
Comment (raubana) [inline gamemodes/attack_of_the_mimics/entities/weapons/swep_aotm_claws.lua]: Again, this was mentioned in the wiki to be less efficient. https://wiki.garrysmod.com/page/table/Random

File: gamemodes/attack_of_the_mimics/gamemode/cl_init.lua [hunk 3]
```diff
@@ -146,6 +146,8 @@ function GM:HUDPaint()
 		-- PROXIMITY SENSE
 		if t == TEAM_MIMIC then
 			drawStat(1,prox_sense,color_black,"PRX")
+			scream_cooldown = (CurTime() - (localplayer:GetActiveWeapon().scream_init or 0)) / localplayer:GetActiveWeapon().Scream.Delay;
```
Comment (raubana) [inline gamemodes/attack_of_the_mimics/gamemode/cl_init.lua:151]: Should be a local variable. This might become global otherwise.

**Review by raubana — CHANGES_REQUESTED** [review]
I'll gladly merge after these minor details are addressed.

**Review by raubana — APPROVED** [review]

**Comment by raubana** [comment]
Oops.

**Comment by raubana** [comment]
I didn't mean to do that. I'm going to add in those requests again.

File: gamemodes/attack_of_the_mimics/entities/weapons/swep_aotm_claws.lua (hunk 1 above)
Comment (raubana) [inline gamemodes/attack_of_the_mimics/entities/weapons/swep_aotm_claws.lua]: This was mentioned in the wiki to be less efficient. https://wiki.garrysmod.com/page/table/Random

File: gamemodes/attack_of_the_mimics/entities/weapons/swep_aotm_claws.lua (hunk 2 above)
Comment (raubana) [inline gamemodes/attack_of_the_mimics/entities/weapons/swep_aotm_claws.lua]: This was mentioned in the wiki to be less efficient. https://wiki.garrysmod.com/page/table/Random

File: gamemodes/attack_of_the_mimics/gamemode/cl_init.lua (hunk 3 above)
Comment (raubana) [inline gamemodes/attack_of_the_mimics/gamemode/cl_init.lua:151]: Should be a local variable. This might become global otherwise.

**Review by raubana — CHANGES_REQUESTED** [review]
Ok, there we go. Once these are addressed I'll merge.

**Comment by FusionLord** [comment]
All requested changes have been made.

**Review by raubana — COMMENTED** [review]
Merged. Thank you.

**Review by raubana — APPROVED** [review]

**Pull request merged**